user_guides/jms/src/main/jbake/content/config.adoc

```diff
@@ -58,7 +58,7 @@ with the necessary configuration steps for your implementation and place
 the file under the `<TS_HOME>/bin/xml/impl/<your_impl>` directory.
 
 For more information, you may wish to view `<TS_HOME>/bin/xml/impl/glassfish/config.vi.xml`,
-the configuration file for Eclipse EE4J Jakarta EE 8 Compatible Implementation, Eclipse GlassFish.
```

Comment:
Hey @edbratt following this discussion https://www.eclipse.org/lists/jakarta.ee-community/msg02415.html I believe that we have to remove the "Eclipse EE4J" before "Jakarta"